src/admin/delete_crate.rs

```diff
@@ -54,5 +55,12 @@ fn delete(opts: Opts, conn: &mut PgConnection) {
         panic!("aborting transaction");
     }
 
-    uploader.delete_index(&client, &krate.name).unwrap();
+    if dotenv::var("FEATURE_INDEX_SYNC").is_ok() {
+        Job::sync_to_git_index(&krate.name).enqueue(conn).unwrap();
```

Comment:
Is it worth having a function that enqueues both the sync jobs?

Reply:
I guess that would make sense :D

Reply:
done :)